lib/PuppeteerSharp/ExecutionContext.cs

```diff
@@ -59,24 +59,13 @@ public ExecutionContext(Session client, ContextPayload contextPayload, Func<dyna
                 return null;
             }
 
-            dynamic remoteObject;
-
-            try
-            {
-                remoteObject = await _client.SendAsync("Runtime.evaluate", new Dictionary<string, object>()
-                {
-                    {"contextId", _contextId},
-                    {"expression", script},
-                    {"returnByValue", false},
-                    {"awaitPromise", true}
-                });
-
-                return ObjectHandleFactory(remoteObject.result);
-            }
-            catch (Exception ex)
+            return await EvaluateHandleAsync("Runtime.evaluate", new Dictionary<string, object>()
```

Comment:
So we're letting the EvaluateHandleAsync fail so we get a more accurate exception?

Reply:
it's the same behavior as the original:
* https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/ExecutionContext.js#L57
* https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/ExecutionContext.js#L69